src/SiteLink.php

```diff
@@ -38,6 +42,11 @@ class SiteLink implements Comparable {
 	private $badges;
 
 	/**
+	 * @var FacetManager
+	 */
+	private $facetManager;
```

Comment:
Where would $facetManager be set to some value? As far as I see it will always be null and not usable in the current implementation of the methods. When would this object be instantiated?